cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/06-school-loans.html

```diff
@@ -0,0 +1,122 @@
+        true,
+        '',
+        'Only borrow what you need. Entering the amount here does not obligate you to borrow this money.'
+    ) }}
+    <div class="block block__mid financial-items">
+        {{
+            input.render({
+                'label': 'Loan from your state',
+                'id': 'loans__stateLoan',
+                'value': '0',
+                'data_attribute': 'data-financial-item="loan_state"'
+            })
+        }}
+        {{
+            input.render({
+                'label': 'Interest rate (%)',
+                'id': 'loans__stateLoanRate',
+                'value': '0',
+                'data_attribute': 'data-financial-item="rate_state"'
+            })
+        }}
+        {{
+            input.render({
+                'label': 'Loan fee (%)',
+                'id': 'loans__stateLoanFee',
+                'value': '0',
+                'data_attribute': 'data-financial-item="fee_stateLoan"'
+            })
+        }}
+        {{
+            input.render({
+                'label': 'Loan from this school',
+                'id': 'loans__schoolLoan',
+                'value': '0',
+                'data_attribute': 'data-financial-item="instiLoan_institutional"'
+            })
+        }}
+        {{
+            input.render({
+                'label': 'Interest rate (%)',
+                'id': 'loans__schoolLoanRate',
+                'value': '0',
+                'data_attribute': 'data-financial-item="rate_institutionalLoan"'
+            })
+        }}
+        {{
+            input.render({
+                'label': 'Loan fee (%)',
+                'id': 'loans__schoolLoanFee',
+                'value': '0',
+                'data_attribute': 'data-financial-item="fee_institutionalLoan"'
+            })
+        }}
+
+        {{
+            input.render({
+                'label': 'Loan from a non-profit',
+                'id': 'loans__nonprofitLoan',
+                'value': '0',
+                'data_attribute': 'data-financial-item="loan_nonprofit"'
+            })
+        }}
+        {{
+            input.render({
+                'label': 'Interest rate (%)',
+                'id': 'loans__nonprofitLoanRate',
+                'value': '0',
+                'data_attribute': 'data-financial-item="rate_nonprofit"'
+            })
+        }}
+        {{
+            input.render({
+                'label': 'Loan fee (%)',
+                'id': 'loans__nonprofitLoanFee',
+                'value': '0',
+                'data_attribute': 'data-financial-item="fee_nonprofit"'
+            })
+        }}
+        {{
+            text_item.render({
+                'label': 'Total loans from your state, this school, or a nonprofit',
+                'value': '$99, 999',
+                'type': 'total',
+                'data_attribute': 'data-financial-item="total_schoolLoans"'
+            })
+        }}
+    </div>
+
+    <div class="costs-not-covered__notification">
```

Comment:
Should this be an element? `costs-not-covered_notification`?

Reply:
Updated to `costs-not-covered-notification` in https://github.com/cfpb/cfgov-refresh/pull/5742/commits/9237cac212c52dfd381dd65958ae6f2af5f8570c.